src/tests/unit/tests/popup/components/diagnostic-view-toggle.test.tsx

```diff
@@ -77,7 +76,7 @@ describe('DiagnosticViewToggleTest', () => {
     });
 
     describe('user interaction: ', () => {
-        it('hanldes click on details view link, it will open fastpass when Assessment enabled', () => {
+        it('handles click on details view link, it will open fast pass when Assessment enabled', () => {
```

Comment:
should it say "FastPass" for consistency with the capitalization in the extension?

Reply:
updated